package.json

```diff
@@ -110,6 +110,7 @@
     "@sindresorhus/slugify": "1.0.0",
     "@statoscope/webpack-plugin": "5.20.1",
     "@storybook/core-server": "7.4.1",
+    "@testing-library/react-hooks": "8.0.1",
```

Comment:
This library shouldn't be required anymore
https://github.com/testing-library/react-hooks-testing-library#a-note-about-react-18-support

And your spec file is actually importing
`import { renderHook } from '@testing-library/react'`

Reply:
Removed ✅